docs/en_US/Overview.md

```diff
@@ -46,6 +46,33 @@ For each experiment, user only needs to define a search space and update a few l
 
 More details about how to run an experiment, please refer to [Get Started](Tutorial/QuickStart.md).
 
+## Core Features
```

Comment:
I'm thinking to replace this or put this ahead of the current "easy-to-use", "scalability", etc. section. let's discuss more in our scrum today.